README.md

```diff
@@ -76,14 +87,18 @@ pipeline step for mabl. This can be written by hand or created via the
 -   continueOnPlanFailure: Continue if your plan/tests fail
 -   environmentId: Selects the environment to run deployments against
     (one of environmentId or applicationId is required)
--   restApiKey: The apiKey of the desired deployment workspace -
+-   restApiKeyName: The name of the API key secret of the desired deployment workspace -
     Required  
-      
+
+*Note* that if you want to select specific plan labels, then use the 
+`labels: ['label1','label2',...,'labeln'].toSet()` syntax in your pipeline step. The Generate Pipeline
+Script page in Jenkins does not append `toSet()`. If you do not use labels, then set the value to null,
+for example: `labels: null`
 
 **Pipeline Step Setup**
 
 ``` syntaxhighlighter-pre
```

Comment:
You can give these sections names. This one is `declerative pipeline`.

Reply:
Yep.